plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/MigrateProcedure.java

```diff
@@ -315,12 +318,18 @@ private Schema toIcebergSchema(List<Column> columns)
         return new Schema(icebergSchema.asStructType().fields());
     }
 
-    private static org.apache.iceberg.types.Type toIcebergType(Type type, AtomicInteger nextFieldId)
+    private static org.apache.iceberg.types.Type toIcebergType(Type type, AtomicInteger nextFieldId, HiveStorageFormat storageFormat)
     {
         if (type instanceof ArrayType || type instanceof MapType || type instanceof RowType) {
             // TODO https://github.com/trinodb/trino/issues/17583 Add support for these complex types
             throw new TrinoException(NOT_SUPPORTED, "Migrating %s type is not supported".formatted(type));
         }
+
+        // TODO (https://github.com/trinodb/trino/pull/17221) Support for Avro
```

Comment:
This PR is already merged. What's the intention of this comment?

Reply:
no intention in such case, deleted